packages/ui/src/common/components/buttons/DetailsButton.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { Meta, Story } from '@storybook/react'
+import React from 'react'
+
+import { DetailsButton, DetailsButtonProps } from '@/common/components/buttons/DetailsButton'
+import { SignOutIcon } from '@/common/components/page/Sidebar/LinksIcons/SignOutIcon'
+
+import { Row, TemplateBlock } from '../storybookParts/previewStyles'
+
+export default {
+  title: 'Common/Buttons/DetailsButton',
+  component: DetailsButton,
+} as Meta
+
+const Template: Story<DetailsButtonProps> = (args) => (
+  <TemplateBlock>
+    <Row>
+      <DetailsButton {...args} />
```

Comment:
To make it the story a bit clearer:
```suggestion
    <BlackBlock>
      <DetailsButton {...args} />
    </BlackBlock>
```
from `../storybookParts/previewStyles`